src/intro/introduction.md

```diff
@@ -41,15 +50,18 @@ This book assumes the following:
     * Interrupts
     * Common interfaces such as I2C, SPI, Serial, etc.
 
-If you are not yet comfortable with Rust, we highly suggest completing the [Rust
-Book] before reading this book.
-
-[Rust Book]: https://doc.rust-lang.org/book/second-edition
-
-If you are not yet comfortable with embedded systems, we highly suggest checking
-out our [other resources] before attempting to read this book.
-
-[other resources]: https://rust-embedded.github.io/bookshelf/
+### Other Resources
+If you are unfamiliar with anything mentioned above or if you want more information about a specific topic mentioned in this book you might find some of these resources helpful.
+
+| Topic        | Resource | Description |
+|--------------|----------|-------------|
+| Rust         | [Rust Book 2018 Edition](https://doc.rust-lang.org/book/2018-edition/index.html) | If you are not yet comfortable with Rust, we highly suggest reading the this book. |
+| Rust         | [Rust Book Second Edition](https://doc.rust-lang.org/book/second-edition) | - |
+| Rust, Embedded | [Embedded Rust Bookshelf](https://rust-embedded.github.io/bookshelf/) | - |
```

Comment:
Please could you also add a link to the [embedded FAQ](https://rust-embedded.github.io/bookshelf/faq.html)?